sys-kernel/bootengine/bootengine-9999.ebuild

```diff
@@ -10,7 +10,7 @@ CROS_WORKON_REPO="git://github.com"
 if [[ "${PV}" == 9999 ]]; then
```

Comment:
How about updating the symlink `bootengine-0.0.38-r6.ebuild` to `bootengine-0.0.38-r7.ebuild`?
The bootengine is notorious for mysterious side-effects caused by garbage cpio files created by previous builds.
So it might be especially a good idea to bump the revision.

Reply:
Sounds good. Will apply that.